internal/compiler/passes/inlining.rs

```diff
@@ -202,6 +202,18 @@ fn inline_element(
     // Now fixup all binding and reference
     for e in mapping.values() {
         visit_all_named_references_in_element(e, |nr| fixup_reference(nr, &mapping));
+
+        // Fix up the named references in inline code of repeated elements that were inlined. Code in bindings
+        // is covered through their corresponding elements being in the mapping, but the init code isn't, hence
+        // this special handling.
+        if e.borrow().repeated.is_some() {
```

Comment:
An alternative would be to extend the mapping data structure with a list of inlined components that need fixing.